backend/__init__.py

```diff
@@ -453,7 +453,7 @@ def add_api_endpoints(app):
     )
     api.add_resource(
         TasksQueriesOwnLockedDetailsAPI,
-        format_url("projects/<int:project_id>/tasks/queries/own/locked/details/"),
+        format_url("projects/tasks/queries/own/locked/details/"),
```

Comment:
I think the `projects/tasks/`isn't fitting that well into our current practice of naming the API. But this is also a special case. I would consider naming it:

* `/api/v2/users/queries/tasks/locked` (similar to the existing `/api/v2/users/queries/favorites/`)
* `/api/v2/projects/all/tasks/queries/own/locked/details/`(includes `all` for the resource)

I kind of prefer the first one. But would also move both api endpoints into the users section. What do you think, @ramyaragupathy ?

Reply:
@xamanu I like `/api/v2/users/queries/tasks/locked`. Another solution would be `/tasks/queries/own/locked/details/`

Reply:
Yes, both would work. But as we have already  another `/api/v2/users/queries/...` so i would go with this one.

Reply:
`/api/v2/users/queries/tasks/locked` seems nifty and consistent with our naming convention. Going with this